minetest.conf.example

```diff
@@ -167,9 +167,9 @@
 
 ### Graphics Effects
 
-#    Makes all liquids opaque
+#    Makes water translucent
```

Comment:
You need not edit this file (`minetest.conf.example`); it is autogenerated from `builtin/settingtypes.txt` when releasing.